docs/codeql/codeql-language-guides/codeql-for-java.rst

```diff
@@ -25,6 +25,7 @@ Experiment and learn how to write effective and efficient queries for CodeQL dat
    javadoc
    working-with-source-locations
    abstract-syntax-tree-classes-for-working-with-java-programs
+   customizing-library-models-for-java
```

Comment:
I _think_ this can be removed if we add a few extra tags to `docs/codeql/codeql-language-guides/customizing-library-models-for-java.rst`. See below.

```suggestion
```

Reply:
I will try it out and see if it works!

Reply:
It builds.